sites/MPIfR/mark5/m5fixtrack/m5fixtrack.c

```diff
@@ -16,17 +16,7 @@
  *   Free Software Foundation, Inc.,                                       *
  *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
  ***************************************************************************/
-//===========================================================================
-// SVN properties (DO NOT CHANGE)
-//
-// $Id: m5fixtrack.c 5105 2013-01-08 09:27:22Z HelgeRottmann $
-// $HeadURL: https://svn.atnf.csiro.au/difx/sites/MPIfR/mark5/m5fixtrack/m5fixtrack.c $
-// $LastChangedRevision: 5105 $
-// $Author: HelgeRottmann $
-// $LastChangedDate: 2013-01-08 17:27:22 +0800 (二, 2013-01-08) $
-//
-//============================================================================
-//
+
```

Comment:
And a blank line here, although I guess this is more in keeping with the usual format